test/E2ETests/Smoke.E2E.Tests/CustomTestExecutionExtensibilityTests.cs

```diff
@@ -40,8 +40,9 @@ public void ExecuteCustomTestExtensibilityWithTestDataTests()
                 "CustomTestMethod2 (B)",
                 "CustomTestMethod2 (B)");
 
+            // TODO :: ASK REVIEW :: SHOULD WE CONTINUE RETURNING A PARENT TEST FOR DATA BOUND TESTS?
```

Comment:
I don't think we should continue to return parent test, as there really is no such concept elsewhere. It was primarily used to support the summary feature which is no longer helpful.